readme.md

```diff
@@ -13,76 +13,100 @@ This API is based on the web-api-skeleton. For more documentation on the skeleto
-```
-+ Use [location-api](https://github.com/osu-mist/locations-api) to fetch `locations-combined.json` and `services.json` by calling `/locations/combined` and `/locations/services`
-+ Post binary data to elasticsearch
+* Install [elasticsearch](https://www.elastic.co/) or run elasticsearch in a docker container:
 
-```bash
-curl -s -XPOST localhost:9200/locations/locations/_bulk --data-binary "@locations-combined.json"; echo
-curl -s -XPOST localhost:9200/services/services/_bulk --data-binary "@services.json"; echo
-```
+  ```
+  $ docker run \
+  -p 9200:9200 \
+  -p 9300:9300 \
+  -e "http.host=0.0.0.0" \
+  -e "transport.host=127.0.0.1" \
+  elasticsearch:2.4.4
+  ```
+
+* Use [location-api](https://github.com/osu-mist/locations-api) to fetch `locations-combined.json` and `services.json` by calling `/locations/combined` and `/locations/services`. You can also get these files from the workspaces of the following Jenkins jobs:
+
+  * [Locations API Backend ES Update (api-dev)](http://act-jenkins.ucsadm.oregonstate.edu:8080/view/APIs/job/apis%20Locations%20API%20Backend%20ES%20Update%20%28api-dev%29)
+  * [Locations API Backend ES Update (api-prod)](http://act-jenkins.ucsadm.oregonstate.edu:8080/view/APIs/job/apis%20Locations%20API%20Backend%20ES%20Update%20%28api-prod%29)
+
+* Post binary data to elasticsearch by using [ES Manager](https://github.com/osu-mist/es-manager):
+
+  ```
+  $ python3 esmanager.py -i locations -t locations locations.json
+  $ python3 esmanager.py -i services -t services services.json
+  ```
 
 ## Generate Keys
 
 HTTPS is required for Web APIs in development and production. Use keytool(1) to generate public and private keys.
 
 Generate key pair and keystore:
 
-    $ keytool \
-        -genkeypair \
-        -dname "CN=Jane Doe, OU=Enterprise Computing Services, O=Oregon State University, L=Corvallis, S=Oregon, C=US" \
-        -ext "san=dns:localhost,ip:127.0.0.1" \
-        -alias doej \
-        -keyalg RSA \
-        -keysize 2048 \
-        -sigalg SHA256withRSA \
-        -validity 365 \
-        -keystore doej.keystore
+```
+$ keytool \
+  -genkeypair \
+  -dname "CN=Jane Doe, OU=Enterprise Computing Services, O=Oregon State University, L=Corvallis, S=Oregon, C=US" \
+  -ext "san=dns:localhost,ip:127.0.0.1" \
+  -alias doej \
+  -keyalg RSA \
+  -keysize 2048 \
+  -sigalg SHA256withRSA \
+  -validity 365 \
+  -keystore doej.keystore
+```
 
 Export certificate to file:
 
-    $ keytool \
-        -exportcert \
-        -rfc \
-        -alias "doej" \
-        -keystore doej.keystore \
-        -file doej.pem
+```
+$ keytool \
+  -exportcert \
+  -rfc \
+  -alias "doej" \
+  -keystore doej.keystore \
+  -file doej.pem
+```
 
 Import certificate into truststore:
 
-    $ keytool \
-        -importcert \
-        -alias "doej" \
-        -file doej.pem \
-        -keystore doej.truststore
+```
+$ keytool \
```

Comment:
Some of these changes are changing the original skeleton readme. Could you update the skeleton repo readme too so things stay in sync?

Reply:
Good point. Will do.